packages/forklift-console-plugin/src/modules/Plans/views/details/tabs/Hooks/PlanHooksSection.tsx

```diff
@@ -0,0 +1,274 @@
+
+    return (
+      <>
+        <Drawer>
+          {canDeleteAndPatchPlanHooks(plan) && (
+            <FlexItem>
+              <Button
+                variant="secondary"
+                icon={<Pencil />}
+                onClick={onToggleEdit}
+                isDisabled={DisableEditHooks}
+              >
+                {t('Edit hooks')}
+              </Button>
+              <HelperText className="forklift-section-plan-helper-text">
+                <HelperTextItem variant="indeterminate">
+                  {DisableEditHooks
+                    ? t(
+                        'The edit hooks button is disabled if the plan started running and at least one virtual machine was migrated successfully.',
+                      )
+                    : t(
+                        'Adding hooks to the plan is optional. Hooks are contained in Ansible playbooks that can be run before or after the migration.',
+                      )}
+                </HelperTextItem>
+                <Divider />
+              </HelperText>
+            </FlexItem>
+          )}
+          {planHooks.length === 0 ? (
+            <HelperText className="forklift-page-plan-details-hooks-msg">
+              <HelperTextItem variant="indeterminate">
+                {t('No hooks have been added to this migration plan.')}
+              </HelperTextItem>
+            </HelperText>
+          ) : (
+            <TableComposable className="forklift-page-plan-details-hooks-section">
+              <Thead>
+                <Tr>
+                  <Th>Hook</Th>
+                  <Th>Migration step</Th>
+                  <Th>Type</Th>
+                  <Th>Definition</Th>
+                </Tr>
+              </Thead>
+              <Tbody>
+                {getPlanHooksList(plan?.spec?.vms, planHooks).map((planHook, i) => (
+                  <Tr key={`${planHook.step}-${i}`}>
+                    <Td>
+                      <ResourceLink
+                        groupVersionKind={HookModelGroupVersionKind}
+                        name={planHook.hook.metadata?.name}
+                        namespace={planHook.hook.metadata?.namespace}
+                      />
+                    </Td>
+                    <Td>
+                      {planHook.step === 'PreHook' ? t(`Pre-migration`) : t(`Post-migration`)}
+                    </Td>
+
+                    <Td>
+                      {planHook.hook.spec?.playbook ? 'Ansible playbook' : 'Custom container image'}
+                    </Td>
+
+                    <Td>
+                      {planHook.hook.spec?.playbook ? (
+                        <Popover
+                          className="forklift-page-plan-details-hooks.playbook-yaml-popover"
+                          aria-label="Playbook YAML contents"
+                          hasAutoWidth
+                          bodyContent={
+                            <SyntaxHighlighter language="yaml" style={a11yLight}>
```

Comment:
we already use patternfly, no need for a new dependency
`@patternfly/react-code-editor`